### ISSUE TITLE
Doesn't work with PHP7

### ISSUE THREAD

**Issue by sprnza** [benjamw/chess#8]
due to mysql_\* functions have been removed in PHP7

**Comment by benjamw** [comment]
Nope, I don't imagine it would. Sorry about that. And I apologize, but I don't have a lot of time to work on it recently, so this issue may be some time before it is resolved properly.

If you want to rectify your local installation, I recommend using https://github.com/philip/MySQLConverterTool as I have had great success with that on many of my conversion projects.

Also... do a regexp search for the following in the PHP files of the script and it will show you where you may need to make some modifications to the variable syntax: (see: http://php.net/manual/en/migration70.incompatible.php#migration70.incompatible.variable-handling)

``` regex
(?:->\s*\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*\s*\[.+\](?:\s*\(.*\))?|::\s*\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*\s*\[.+\]\s*\(.*\)|\$\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*\s*(?:\[|\())
```

(Note: I searched my local installation of WebChess for that regexp and found nothing, but it's good to know when updating scripts to PHP7)
